### DISCUSSION TITLE
Command on quit

### DISCUSSION THREAD

**Discussion by noahfraiture** [zellij-org/zellij#3514] [Q&A] [answered]
Hello, I have a layout that open different shell inside a container and I would like to close that container when I quit the layout. Is it possible to run a command when I quit the zellij layout with ctrl+q ? I use exegol

**Comment by Lockszmith-GH** [comment, marked as answer]
I can think of a number of ways to deal with this.

One is to map the <kbd>Ctrl</kbd>+<kbd>q</kbd> to run a command, then exit.

Another is to have the layout executable be `sh -c "<execute container>; <execute cleanup>"` - this way when you exit the container, the cleanup action runs.
